data/divyashaktipal.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Divyashakti Pal",
+  "type": "personal",
+  "bio": "Student "
```

Comment:
The JSON is invalid, You need to add a comment after "bio" property.

```json
"bio": "student",
```

Reply:
I have corrected this